koin-projects/koin-core/src/main/kotlin/org/koin/core/KoinComponent.kt

```diff
@@ -54,6 +67,19 @@ inline fun <reified T> KoinComponent.inject(
 ): Lazy<T> =
     lazy(LazyThreadSafetyMode.NONE) { getKoin().get<T>(qualifier, parameters) }
 
+/**
+ * Lazy inject a Koin instance if available
+ * @param qualifier
+ * @param parameters
+ *
+ * @return Lazy instance of type T or null
+ */
```

Comment:
Need to add @KoinApiExtension

Reply:
Since it was developed based on the master branch before, I didn't notice this part. Now it has been corrected, thanks.